src/app/components/views/job-view/job-view.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { JobData } from 'src/app/models/job-data.model';
+import { JobService } from 'src/app/services/job.service';
+
+@Component({
+  selector: 'app-job-view',
+  templateUrl: './job-view.component.html',
+  styleUrls: ['./job-view.component.scss']
+})
+export class JobViewComponent implements OnInit {
+  public job: JobData; // To be retrieved from jobService
+  public activeTab: number = 0;
+  public readonly tabTitleRoutes: string[] = ['job', 'customer', 'history', 'facilities', 'tests'];
+
+  constructor(private jobService: JobService, private router: Router) { }
+
+  ngOnInit(): void {
+    this.job = this.jobService.getSelectedJob(); // Will be undefined if no job selected
```

Comment:
can you move this functionality out of oninit?

Reply:
Moved to specific methods for each of the two tasks